core/storage/user/gae_models.py

```diff
@@ -2235,9 +2235,9 @@ def get_deletion_policy():
 
     @staticmethod
     def get_export_policy():
-        """Model does not need to exported as it hold authentication details
-        relavant only for backend, and does not contain any exportable
-        user data.
+        """Currently the model hold authentication details relavant only for
```

Comment:
``Currently, the model holds authentication...``

Reply:
Done.